src/__tests__/db.integration.test.js

```diff
@@ -2,86 +2,114 @@ import { v4 as uuid } from 'uuid';
 import ModelFactory from '../models';
 import { modelName as messageFragmentModel } from '../models/message-fragment';
 import { modelName as registrationRequestModel } from '../models/registration-request';
+import { modelName as messageIdReplacementModel } from "../models/message-id-replacement";
 import { readFile } from './utilities/integration-test.utilities';
-import { patientAndPracticeOdsCodesMatch, updateAllFragmentsMessageIds } from '../services/transfer/transfer-out-util';
-import { getAllFragmentsWithMessageIdsFromRepo } from '../services/ehr-repo/get-fragments';
+import {
+  updateFragmentMessageId
+} from '../services/transfer/transfer-out-util';
+import {
+  getFragment,
+  retrieveIdsFromEhrRepo
+} from '../services/ehr-repo/get-fragment';
 import { transferOutFragments } from '../services/transfer/transfer-out-fragments';
-import { sendFragment } from '../services/gp2gp/send-fragment';
 import { createRegistrationRequest } from '../services/database/create-registration-request';
 import { logger } from "../config/logging";
+import { createMessageIdReplacement } from "../services/database/create-message-id-replacement";
+import nock from "nock";
```

Comment:
this nock import looks out of place without the curly brackets _{ nock }_

Reply:
Hey, so we're not able to import nock as `{ nock }` as it is a default export!